app/helpers/hyku_addons/cross_tenant_shared_search_helper.rb

```diff
@@ -4,28 +4,23 @@
 module HykuAddons
   module CrossTenantSharedSearchHelper
     def generate_work_url(model, request)
+      # returns a symbol for id and string for other attributes
       model = model.with_indifferent_access
```

Comment:
It look like you have this here so that you can use `model[:id]`, but I can't see anywhere else where it would be used. Could we remove it and then access the ID via `model["id"]`?